tests/test_very_basic_operations.py

```diff
@@ -18,11 +19,11 @@ def test_equality(self):
         self.assertEqual(factorial(5), 120)
 
     def test_argument_float(self):
-        with self.assertRaises(ValueError):
+        with self.assertRaises(ValueError, msg = 'Number is not integral'):
```

Comment:
First, correctly is `msg='Number is not integral'`, without spaces around =

Second, assertRaises checks that exception is just raised, you can use assertRaisesRegex(ValueError, some_text) to check that exception is raised and text of exception is the same as `some_text`. You will need this because I want you to raise two ValueError exceptions with different text, so you'll be able to check that correct text is in exception